ui/pages/confirmations/confirmation/templates/__snapshots__/add-ethereum-chain.test.js.snap

```diff
@@ -17,12 +17,14 @@ exports[`add-ethereum-chain confirmation should match snapshot 1`] = `
           <div
             class="mm-box mm-text mm-avatar-base mm-avatar-base--size-xs mm-avatar-network mm-picker-network__avatar-network mm-text--body-xs mm-text--text-transform-uppercase mm-box--display-flex mm-box--justify-content-center mm-box--align-items-center mm-box--color-text-default mm-box--background-color-background-alternative mm-box--rounded-full mm-box--border-color-transparent box--border-style-solid box--border-width-1"
           >
-            ?
+            T
           </div>
           <span
             class="mm-box mm-text mm-text--body-sm mm-text--ellipsis mm-box--color-text-default"
             data-testid="network-display"
-          />
+          >
+            Test initial state
```

Comment:
The test fixtures used here were of the type affected by this change: `providerConfig` was not a built-in or custom network. Probably not intentional, but I didn't dig in further. This snapshot change is not wrong or totally unexpected at least.